scripts/run-ios.sh

```diff
@@ -39,41 +39,53 @@ if [ -z "${1-}" ]; then
     exit 1
 fi
 
-SIMULATOR=${1}
+# fetch available iOS Simulators
+xcrun simctl list devices -j > ios_simulators_list.json
 
-# get our desired UUID
-UUID=$(xcrun simctl list devices | grep -E "$SIMULATOR \(" | head -n 1 | awk -F '[()]' '{print $2}')
+SIMULATOR=${1}
```

Comment:
You're still not checking if this is set. If not it should fail right?

Reply:
we do that -> https://github.com/status-im/status-mobile/blob/9068887ad3bf87d77c9e6bd300f0d8eef4da03d9/scripts/run-ios.sh#L36-L40

Reply:
Oh, got folded up, my mistake.